web/client/codechecker_client/cmd/store.py

```diff
@@ -347,37 +320,268 @@ def collect_file_hashes_from_plist(plist_file):
+        replace_report_hash(plist_file, HashType.CONTEXT_FREE)
+        return True
+    return False
+
+
+def get_report_data(reports):
+    """Return the minimal required report information to be able
+    to collect review comments from the source code.
+    """
+    report_main = []
+    for report in reports:
+        last_report_event = report.bug_path[-1]
+        file_path_index = last_report_event['location']['file']
+        report_line = last_report_event['location']['line']
+        report_main.append(ReportLineInfo(report_line,
+                                          file_path_index,
+                                          ""))
+    return report_main
+
+
+def scan_for_review_comment(job):
+    """Scan a file for review comments returns
+    all the found review comments.
+    """
+    file_path, lines = job
+    sc_handler = SourceCodeCommentHandler()
+    comments = []
+    with open(file_path, mode='r',
+              encoding='utf-8',
+              errors='ignore') as sf:
+        comments, misspelled_comments = \
+                sc_handler.scan_source_line_comments(sf, lines)
+
+        if misspelled_comments:
+            LOG.warning("There are misspelled review status comments in %s",
+                        file_path)
+        for mc in misspelled_comments:
+            LOG.warning(mc)
+
+    return comments
+
+
+def filter_source_files_with_comments(source_file_info, main_report_positions):
+    """Collect the source files where there is any codechecker review
+    comment at the main report positions.
+    """
+
+    files_with_comment = set()
+    jobs = []
+    for file_path, v in source_file_info.items():
+        if not bool(v):
+            # missing file
+            continue
+        lines = [rep.line for rep in main_report_positions
+                 if rep.filepath == file_path]
+
+        jobs.append((file_path, lines))
+
+    with concurrent.futures.ProcessPoolExecutor() as executor:
+        for job, comments in zip(jobs,
+                                 executor.map(scan_for_review_comment, jobs)):
+            file_path, _ = job
+            if comments:
+                files_with_comment.add(file_path)
+
+    return files_with_comment
+
+
+def parse_collect_plist_info(plist_file):
+    """Parse one plist report file and collect information
+    about the source files mentioned in the report file.
+    """
+
+    source_files, reports = parse_report_file(plist_file)
+
+    if len(source_files) == 0:
+        # If there is no source in the plist we will not upload
+        # it to the server.
+        LOG.debug("Skip empty plist file: %s", plist_file)
+        rli = ReportFileInfo(store_it=False, main_report_positions=[])
+        sfir = SourceFilesInReport(source_info={},
+                                   missing=set(),
+                                   changed_since_report_gen=set())
+        return rli, sfir
+
+    missing_files = set()
+
+    # What if some file was updated?
```

Comment:
Is this comment still relevant? If yes, maybe we can highlight it with TODO or FIXME tags.